nat-lab/tests/test_notification_center.py

```diff
@@ -64,7 +66,7 @@ async def test_nc_register():
     async with AsyncExitStack() as exit_stack:
         # Setup connections
         connection_beta = await exit_stack.enter_async_context(
```

Comment:
For this test, I do not think you need client container, you should be able to do it from the test itself directly.